src/components/BlueContainer/BlueContainer.module.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../style/default.scss';

```

Comment:
You should use `@use` instead. `@import` is still widely used, but it is inferior

Read:

https://sass-lang.com/documentation/at-rules/import